cypress/e2e/settings/circulation/fixed-due-date-schedules/schedule-does-not-cover-the-test-date.cy.js

```diff
@@ -78,8 +76,9 @@ describe('ui-circulation-settings: Fixed due date schedules', () => {
           departments: [],
         })
           .then((user) => {
-            userId = user.id;
-            userBarcode = user.barcode;
+            userData.userId = user.id;
+            userData.userBarcode = user.barcode;
+            userData.userLastName = user.lastName;
```

Comment:
you can use spread operator
`userData = [ ...user ]`

Reply:
done